dist/index.html

```diff
@@ -3,14 +3,20 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fitness</title>
     <link
+      href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;600;700;800&display=swap"
       rel="stylesheet"
-      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"
-      integrity="sha256-h20CPZ0QyXlBuAw7A+KluUYx/3pK+c7lYEpqLTlxjYQ="
-      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css"
+    />
+    <link
```

Comment:
I think this is a duplicate file, please remove it otherwise the page will request this file twice each time someone visits us.

Reply:
No, one is .css and the other is .js both are required...

Reply:
Then two things: 
- <link> tag doesn't work with `.js` files
- why do we need `js` for font-awesome at all?

Reply:
removed